crypto-ffi/bindings/jvm/src/test/kotlin/com/wire/crypto/MLSTest.kt

```diff
@@ -387,3 +391,30 @@ fun randomIdentifier(n: Int = 12): String {
         .map { kotlin.random.Random.nextInt(0, charPool.size).let { charPool[it] } }
         .joinToString("")
 }
+
+interface DeliveryService : MlsTransport {
+    suspend fun getLatestCommitBundle(): CommitBundle
+
+    suspend fun getLatestWelcome(): Welcome
+
+    suspend fun getLatestCommit(): MlsMessage
+}
+
+class  MlsTransportSuccessProvider : DeliveryService {
```

Comment:
```suggestion
class  MockMlsTransportSuccessProvider : DeliveryService {
```

Let's make it clear that this is mock implementation which we are using in the tests.